mesonbuild/mlog.py

```diff
@@ -409,8 +412,15 @@ def exception(self, e: Exception, prefix: T.Optional[AnsiDecorator] = None) -> N
                     node_resolve.append(empty_lineno + ' | ' + colno * ' ')
                     node_resolve.append(yellow(f'{error_resolve}'))
 
+                if lineno != end_lineno:
+                    node_after += [['...located before...']]
+                    node_after += [
+                        [f' {str(i + lineno + 1).rjust(lineno_size)} | ' + line]
+                        for i, line in enumerate(source_lines[1:])
+                    ]
+
```

Comment:
I think using the `.extend()` method would make this clearer: `node_after.extend([...])` vs `node_after += [[...]]`

Reply:
This would be `node_after.extend([[...]])`  since node_after is a list (which gets unpacked) where each item is one of the line for the output.

Reply:
Ah, you're right. Never mind on this one then.